src/main/resources/templates/pps/bankTransferSanctionsDetails.html

```diff
@@ -0,0 +1,48 @@
+<!DOCTYPE html>
+
+<html xmlns:th="http://www.thymeleaf.org"
+      xmlns:layout="http://www.ultraq.net.nz/thymeleaf/layout"
+      layout:decorate="~{layouts/baseLayout}">
+
+<head>
+    <title>
+        Pay by bank transfer - Confirmation statement
+    </title>
+</head>
+
+<div id="bank-transfer-confirmation-statement-main-content" layout:fragment="content">
```

Comment:
Can the 2 ids here be changed to get rid of the `confirmation-statement` stuff.
Maybe something like `bank-transfer-sanctions-details-main-content` which matches the protoype addess bar and your page filename more closely in kebab case?